google/ggrc-core
# GGRC-8050 GCA with long titles without spaces incorrectly displayed in the filter

# Issue description

The list with Attributes looks broken, long title does not wrap to the next line.

# Steps to test the changes

1. Login as admin in GGRC
2. Open Administration page => Custom Attributes tab
3. Open Assessment
4. Add GCA with long title without spaces in the title
5. Open Global Search
6. Select Object type = Assessment
7. Click on the field with objects (Title is displayed by default)
8. Scroll down and find created in the 4th step GCA with a long title
9. Expected: The list with Attributes should not be broken, title should wrap to the next line

# Solution description

Added word-break property for autocomplete-dropdown

# Sanity checklist




# PR Review checklist
